src/main/webapp/creatives/maps/simid_map_creative.js

```diff
@@ -11,6 +11,10 @@ const AdParamKeys = {
 const FIND_NEAREST_TEMPLATE_TEXT = "Find Nearest ";
 const DEFAULT_BUTTON_LABEL = "Location";
 const DEFAULT_ZOOM = 13;
+let map;
```

Comment:
Prefer member variables of the class to global variables. Unless there's something I'm missing that should make these be global.

One reason is for unit testing. Member variables can be reset between tests by making a new class but global variables might have unwanted behaviors.  Member variables (ie. this.map) reduce visibility too and help make fewer bugs.

Reply:
Fixed